### DISCUSSION TITLE
I would like to ask a question about the dpgen simplify function

### DISCUSSION THREAD

**Discussion by xwxwshi** [deepmodeling/dpgen#1560] [Q&A]
### Summary

Question about dpgen simplify, after running it, will he remove the duplicates or will he output a new database?

### DP-GEN Version

0.12.0

### Platform, Python Version, etc

Bohrium

### Details

I'd like to ask if the data I simplified after running it, will the duplicates be removed or will I be re-categorized for me

**Comment by njzjz** [comment]
`dpgen simplify` uses the same theory as `dpgen run`. The only difference thing is that the source data of `dpgen run` is the simulation trajectory using developing models, while the source data of `dpgen simplify` is the existing data set using any strategy.
